near-sdk/src/collections/tree_map.rs

```diff
@@ -16,54 +16,37 @@ use crate::collections::UnorderedMap;
 #[derive(BorshSerialize, BorshDeserialize)]
 pub struct TreeMap<K, V> {
     root: u64,
-    val: UnorderedMap<K, V>,
+    val: LookupMap<K, V>,
     tree: Vector<Node<K>>,
 }
 
 #[derive(Clone, BorshSerialize, BorshDeserialize)]
 pub struct Node<K> {
     id: u64,
-    key: K,             // key stored in a node
-    lft: Option<u64>,   // left link of a node
-    rgt: Option<u64>,   // right link of a node
-    ht: u64,            // height of a subtree at a node
+    key: K,           // key stored in a node
+    lft: Option<u64>, // left link of a node
+    rgt: Option<u64>, // right link of a node
+    ht: u64,          // height of a subtree at a node
 }
 
 impl<K> Node<K>
-    where
-        K: Ord + Clone + BorshSerialize + BorshDeserialize
+where
+    K: Ord + Clone + BorshSerialize + BorshDeserialize,
 {
     fn of(id: u64, key: K) -> Self {
-        Self {
-            id,
-            key,
-            lft: None,
-            rgt: None,
-            ht: 1,
-        }
-    }
-}
-
-impl<K, V> Default for TreeMap<K, V>
-    where
-        K: Ord + Clone + BorshSerialize + BorshDeserialize,
-        V: BorshSerialize + BorshDeserialize,
-{
-    fn default() -> Self {
-        Self::new(next_trie_id())
+        Self { id, key, lft: None, rgt: None, ht: 1 }
```

Comment:
Maybe left, right and height would be more readable?

Reply:
Yep. Bad naming.